specs/core/0_beacon-chain.md

```diff
@@ -898,10 +916,10 @@ def get_shuffled_index(index: ValidatorIndex, index_count: int, seed: Bytes32) -
     return index
 ```
 
-### `compute_committee`
+### `get_committee_indices`
 
 ```python
-def compute_committee(indices: List[ValidatorIndex], seed: Bytes32, index: int, count: int) -> List[ValidatorIndex]:
+def get_committee_indices(indices: List[ValidatorIndex], seed: Bytes32, index: int, count: int) -> List[ValidatorIndex]:
```

Comment:
This doesn't take a `BeaconState` input, so I would argue it is a `compute_` rather than `get_`

Reply:
Some time ago I argued along the same lines, i.e. reserve `get_` for functions that get state from `BeaconState`. I vaguely remember HW and V arguing for a laxer use of `get_` for pure functions.

I don't care either way so long as we're consistent. Given that `get_epoch_start_slot`, `get_shuffled_index`, `get_committee_indices`, `get_bitfield_bit`, `get_delayed_activation_exit_epoch` and `get_genesis_beacon_state` all have the `get_` prefix without taking a `BeaconState` input, I think the path of least resistance is to go with `get_` for pure functions :)